echopype/echodata/convention/echodata_structures.py

```diff
@@ -0,0 +1,402 @@
+"""
+The purpose of this dictionary is to define the structure of each sensor
+across echopype versions. The dictionary was originally intended to ease
+the transition from echopype version 0.5.x to version 0.6.x. This dict
+primarily focuses on names each EchoData group and their coordinates.
+The structure is currently based on the convention file 1.0.yml.
```

Comment:
*Small edits:*

across echopype versions. The dictionary was **developed** ~~originally intended~~ to ease
the transition from echopype version 0.5.x to version 0.6.x. This dict
primarily focuses on names **of** each EchoData group and their coordinates.
The structure is currently based on the **SONAR-netCDF4** convention file 1.0.yml.